x/accounts/proto/cosmos/accounts/defaults/lockup/tx.proto

```diff
@@ -79,6 +79,17 @@ message MsgUndelegate {
   cosmos.base.v1beta1.Coin amount            = 3 [(gogoproto.nullable) = false, (amino.dont_omitempty) = true];
 }
 
+// MsgWithdrawReward defines a message that enable lockup account to execute withdraw reward message
```

Comment:
has this been implemented?

Reply:
yes it is being implemented and registered in base lockup